packages/winpanda/extra/src/winpanda/core/cmdconf.py

```diff
@@ -102,6 +102,11 @@ def __init__(self, **cmd_opts):
         self.dcos_conf = self.get_dcos_conf()
         LOG.debug(f'{self.msg_src}: dcos_conf: {self.dcos_conf}')
 
+        # Add discovery configuration for dcos_conf
+        discovery_type = self.dcos_conf.get('values').get(
+            'master_discovery', 'master_http_loadbalancer')
+        self.cluster_conf['discovery'] = {'type': discovery_type}
```

Comment:
Is there any reason to store a dictionary rather than the string?

Reply:
eventualy it will be converted in object with property value so string will not work here, I have already try this